compiler/qsc_formatter/src/formatter.rs

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use qsc_data_structures::span::Span;
+use qsc_frontend::lex::{
+    concrete::{self, ConcreteToken, ConcreteTokenKind},
+    cooked::TokenKind,
+    Delim,
+};
+
+#[cfg(test)]
+mod tests;
+
+#[derive(Debug)]
+pub struct Edit {
```

Comment:
See comment in protocol.rs where TextEdit was added with the fields 'span' and 'contents'. Should we have one struct here, consistent with the VS Code API struct if possible?

Reply:
We can have a struct here that looks just like the one in the protocol.rs, but the language service package is further down the dependency stream, so the formatter mod can't take a dependency on it to use that TextEdit struct.